html/rendering/non-replaced-elements/the-hr-element-0/hr.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<title>The hr element</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#ref {
+  display: block;
+  unicode-bidi: isolate;
```

Comment:
It's not a spec bug, technically. Just a bit confusing if you assume a single section mentions everything that about the feature the section is about. But the spec would balloon in size if that was consistently done for everything.